data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -1201,7 +1202,7 @@
     "armor_bash": 4,
     "armor_cut": 4,
     "armor_bullet": 6,
-    "hp": 350,
+    "hp": 335,
     "zombify_into": "mon_zosasaurus",
     "description": "A huge aquatic dinosaur about the size of a small house, with a ferocious crocodile-like head and powerful swimming fins.",
```

Comment:
They're dinosaurs in specific? The zinosaur_burned.json file described them as an aquatic reptile.

Reply:
Fixed, thanks for catching that